src/Bluejay.asm

```diff
@@ -340,11 +339,8 @@ Pgm_LED_Control: DS 1                   ; LED control
 Pgm_Power_Rating: DS 1                  ; Power rating
 Pgm_Safety_Arm: DS  1                   ; Various flag settings: bit 0 is require edt enable to arm
 
-ISEG AT 0B0h
-Stack: DS 16                            ; Reserved stack space
-
 ISEG AT 0C0h
-Dithering_Patterns: DS 16               ; Bit patterns for pwm dithering
+Stack: DS 16                            ; Reserved stack space
```

Comment:
Maybe we should add a comment that it is reserved, but can be used?

Reply:
I moved the stack to the former Dithering_Pattterns space. There is no problem for moving the stack space